Assets/Scripts/Ability/AbilityManager.cs

```diff
@@ -87,21 +96,60 @@ private void OnDestroy()
         private void OnAbility1(InputAction.CallbackContext obj)
         {
             HandleAbilityBySlot((int)AbilitySlot.First);
+            _indicatorController.enabled = false;
```

Comment:
Why is this not a function?

Reply:
didn't want to bother but it is done now